ui/component/txoList/view.jsx

```diff
@@ -0,0 +1,264 @@
+          } else {
+            newUrlParams.set(TXO.SUB_TYPE, 'all');
+          }
+        }
+        if (currentUrlParams.active) {
+          newUrlParams.set(TXO.ACTIVE, currentUrlParams.active);
+        }
+        newUrlParams.set(TXO.PAGE, String(1));
+        newUrlParams.set(TXO.PAGE_SIZE, currentUrlParams.pageSize);
+        break;
+      case TXO.SUB_TYPE:
+        if (currentUrlParams.type) {
+          newUrlParams.set(TXO.TYPE, currentUrlParams.type);
+        }
+        if (currentUrlParams.active) {
+          newUrlParams.set(TXO.ACTIVE, currentUrlParams.active);
+        }
+        newUrlParams.set(TXO.SUB_TYPE, delta.value);
+        newUrlParams.set(TXO.PAGE, String(1));
+        newUrlParams.set(TXO.PAGE_SIZE, currentUrlParams.pageSize);
+        break;
+      case TXO.ACTIVE:
+        if (currentUrlParams.type) {
+          newUrlParams.set(TXO.TYPE, currentUrlParams.type);
+        }
+        if (currentUrlParams.subtype) {
+          newUrlParams.set(TXO.SUB_TYPE, currentUrlParams.subtype);
+        }
+        newUrlParams.set(TXO.ACTIVE, delta.value);
+        newUrlParams.set(TXO.PAGE, String(1));
+        newUrlParams.set(TXO.PAGE_SIZE, currentUrlParams.pageSize);
+        break;
+    }
+    return `?${newUrlParams.toString()}`;
+  }
+
+  const paramsString = JSON.stringify(params);
+
+  useEffect(() => {
+    if (paramsString && fetchTransactionPage) {
+      const params = JSON.parse(paramsString);
+      fetchTransactionPage(params);
+    }
+  }, [paramsString, fetchTransactionPage]);
+
+  return (
+    <Card
+      title={__(`Transactions`)}
+      titleActions={
+        <div className="card__actions--inline">
+          <Button button="secondary" label={__('Refresh')} onClick={() => fetchTransactionPage(params)} />
+          {/* @if TARGET='app' */}
+          <Button button="primary" navigate={`/$/${PAGES.TRANSACTIONS}`} label={__('Full History')} />
+          {/* @endif */}
+        </div>
+      }
+      headerActions={
+        <div className="card__actions--between">
+          <div>
+            <div className="card__actions">
+              {/* show the main drop down */}
+              <div>
+                <FormField
+                  type="select"
+                  name="type"
+                  label={__('Type')}
+                  value={type || 'all'}
+                  onChange={e => handleChange({ dkey: TXO.TYPE, value: e.target.value })}
+                >
+                  {Object.values(TXO.DROPDOWN_TYPES).map(v => {
+                    const stringV = String(v);
+                    return (
+                      <option key={stringV} value={stringV}>
+                        {stringV && __(toCapitalCase(stringV))}
+                      </option>
+                    );
+                  })}
+                </FormField>
+              </div>
+              {/* show the subtypes drop down */}
+              {(type === TXO.SENT || type === TXO.RECEIVED) && (
+                <div>
```

Comment:
There are a few extra divs in this component

Reply:
I hit some nth of type top and side padding in FormField unless I wrapped in div.